### PR TITLE
Reports: Eliminate the possibility of overflow

### PR DESCRIPTION
**Pull request by qzhjason**
When test has very large sum number.

### CODE REVIEW

**Comment by slandelle** [comment]
@qzhjason Nice catch, thanks!

Then, your fix doesn't compile (unmatched parens) and it would probably be better to have a double mean too.
Can you please fix and rebase your commit?

**Comment by qzhjason** [comment]
I'll try to fix it on my local PC.

File: gatling-charts/src/main/scala/io/gatling/charts/stats/buffers/GeneralStatsBuffers.scala [hunk 1]
```diff
@@ -84,8 +84,8 @@ private[stats] class GeneralStatsBuffer(durationInSec: Long) {
 
     } else {
       val count = digest.size
-      val mean = (sum / count).toInt
-      val stdDev = math.sqrt((sumOfSquares - (sum * sum) / count) / count).toInt
+      val mean = (sum / count).toDouble
```
Comment (slandelle) [inline gatling-charts/src/main/scala/io/gatling/charts/stats/buffers/GeneralStatsBuffers.scala]: This doesn't do much. You want to turn sum into a double, then divide.

File: gatling-charts/src/main/scala/io/gatling/charts/stats/buffers/GeneralStatsBuffers.scala [hunk 2]
```diff
@@ -84,8 +84,8 @@ private[stats] class GeneralStatsBuffer(durationInSec: Long) {
...
+      val stdDev = math.sqrt(sumOfSquares / count - mean * mean).toInt
```
Comment (slandelle) [inline gatling-charts/src/main/scala/io/gatling/charts/stats/buffers/GeneralStatsBuffers.scala]: sumOfSquares.toDouble

**Comment by slandelle** [comment]
Could you please squash your commits into a single one?

**Pull request closed without merging**